source/isaaclab_assets/isaaclab_assets/robots/kuka.py

```diff
@@ -13,7 +13,7 @@
 
 KUKA_KR210_CFG = ArticulationCfg(
     spawn=sim_utils.UsdFileCfg(
-        usd_path="/workspace/isaaclab/usd/robots/kuka/kr210/kr210.usd",
+        usd_path="/workspace/isaaclab/usd/robots/kuka/kr210/kr210_cutter.usd",
```

Comment:
should we leave the kr210 and perhaps in comment or complete remove the cutter? given that that's a quite custom asset